client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -30,6 +34,28 @@ interface Props {
 
 const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const hardCodedCredential = {
```

Comment:
Let's stores the credentials in a separate file and import it in both forms as opposed to redeclaring this variable here